src/wasm-type.h

```diff
@@ -432,10 +434,8 @@ struct Array {
 };
 
 struct Rtt {
-  enum {
-    // An Rtt can have no depth specified
-    NoDepth = -1
-  };
+  // An Rtt can have no depth specified
+  static constexpr uint32_t NoDepth = -1;
```

Comment:
I'm fine either way, but what is the benefit of this approach? The type?

Reply:
Yeah, the type explicitly matches its uses, the code is slightly shorter, and it's consistent with other magic values defined in this file (i.e. `_last_basic_type`).